### DISCUSSION TITLE
Switch phases / Phasenumschaltung

### DISCUSSION THREAD

**Discussion by BadP1t** [cathiele/homeassistant-goecharger#89] [Ideas]
Hi,
thanks @cathiele for your great integration.
Is it possible to change phases?
I will soon be looking into automating the EV-Charger for PV excess, and therfore it would be appreciated.

**Comment by krenlund** [comment]
Hi, I second this request. It would be very handy when automating the charging to follow solar output, when three phases are too much even with 6A.

**Comment by doebs** [comment]
It would be very handy, to manualy select the number of phases manualy within this integration as switch.
Until then we could use http wir parameters PSM=1 (1 phase), PSM=2 (3 phases).

**Comment by kangxidadide** [comment]
Me too.

**Comment by jmtatsch** [comment]
I am now using this [integration](https://github.com/syssi/homeassistant-goecharger-mqtt) which allows flawless PV-surplus charging with automatic phase switching
